docs/docs/migration/v0.25.0.md

```diff
@@ -64,4 +64,934 @@ ignite s module mars
 
 The only difference is the additional directory `planet` which is the name of the application.  The name of the app can 
 be verified by checking the package in the `go.mod` file.  In this example, the package is `github.com/cosmos/planet` where
-`planet` is the app name.
\ No newline at end of file
+`planet` is the app name.
+
+ ---
+
+## Removing `cosmoscmd`
+
+`v0.25.0` removes the `cosmoscmd` package from scaffolded chains.  This package provided  utility for creating 
+commands and starting up their application.  The `cosmoscmd` package is now deprecated, and it is suggested that chains
+import this functionality in their codebase so that their chains can be more easily upgraded and customized.
+
+The main functionality of `cosmoscmd` will be moved to the `app` package of your chain.  Some imports in these 
+examples contain the sample string, `{ModulePath}`.  Replace this string with the Go module path of your blockchain.
+For example, if your blockchain module path is `github.com/planet/mars`, `{ModulePath}/app/params` would be become
+`github.com/planet/mars/app/params`.
+
+#### Migration in `app` package
+
+To begin, create  a new file, `./app/params/encoding.go`, containing the following code:
```

Comment:
There's no reference to create a new file in `.app/params/amino.go` 
https://github.com/cosmos/gaia/blob/main/app/params/amino.go

Should be really necessary this inclusion or not?